onchain/src/social/account.cairo

```diff
@@ -48,8 +65,12 @@ pub mod SocialAccount {
         ) { // TODO: implement handle transfer logic
         }
     }
-}
 
+    #[external(v0)]
+    fn supports_interface_fn(self: @ContractState, interface_id: felt252) -> bool {
```

Comment:
Why `fn` at the end of a function?

Reply:
Oh my bad, It was an oversight.
I have given it a better name.